cmd/internal/planetscale_edge_database.go

```diff
@@ -281,22 +287,27 @@ func (p PlanetScaleEdgeDatabase) sync(ctx context.Context, tc *psdbconnect.Table
 		keyspaceOrDatabase = ps.Database
 	}
 
+	isFullSync := syncMode == "full"
+	copyCompletedSeen := false
 	// Can finish sync once we've synced to the stop position, or finished the VStream COPY phase
 	canFinishSync := false
 	resultCount := 0
 
-	var fields []*query.Field
-
 	for {
 		res, err := c.Recv()
```

Comment:
@mattlord I found that on the "no more VGTIDs" case, the process was blocking here. So I opted to just keep the regular `context.WithTimeout` (instead of a timer like we discussed), but increased the wait time from 1 minute to 5 minutes.

After testing locally on a similarly higher traffic table & database, 5 minutes was more successful than the 1 minute timeout.